protoc-gen-openapiv2/internal/genopenapi/types.go

```diff
@@ -152,10 +152,11 @@ type openapiParameterObject struct {
 // core part of schema, which is common to itemsObject and schemaObject.
 // http://swagger.io/specification/#itemsObject
 type schemaCore struct {
-	Type    string          `json:"type,omitempty"`
-	Format  string          `json:"format,omitempty"`
-	Ref     string          `json:"$ref,omitempty"`
-	Example json.RawMessage `json:"example,omitempty"`
+	Type     string          `json:"type,omitempty"`
+	Format   string          `json:"format,omitempty"`
+	Ref      string          `json:"$ref,omitempty"`
+	Nullable bool            `json:"x-nullable,omitempty"`
```

Comment:
`x-nullable` isn't defined in the referenced docs (http://swagger.io/specification/#itemsObject), could you please add a reference here?

Reply:
I updated that URL to point to the v2 specification, and added a reference to the v3 spec that calls out `nullable`. `x-nullable` is a "back-ported" extension to emulate this v3 behavior in v2, but it's more of a widely used convention than part of any formal spec. I've added what I think is helpful; let me know if it's too much or missing something you'd want to see/know.

Reply:
I updated that URL to point to the v2 specification, and added a reference to the v3 spec that calls out nullable. x-nullable is a "back-ported" extension to emulate this v3 behavior in v2, but it's more of a widely used convention than part of any formal spec. I've added what I think is helpful; let me know if it's too much or missing something you'd want to see/know.